trace-utils/src/trace_utils.rs

```diff
@@ -31,63 +28,21 @@ macro_rules! parse_string_header {
     }
 }
 
-#[derive(Debug, Deserialize, Serialize, Clone, PartialEq)]
-pub struct Span {
```

Comment:
no longer needed, as the pb::Span struct now has serde default annotations on certain fields allowing for direct deserialization of requests into them